src/modules/hunting/etcd.py

```diff
@@ -0,0 +1,146 @@
+import json
+import logging
+
+import requests
+
+from ...core.events import handler
+from ...core.events.types import Vulnerability, Event, OpenPortEvent
+from ...core.types import ActiveHunter, Hunter, KubernetesCluster, InformationDisclosure, RemoteCodeExec, UnauthenticatedAccess, AccessRisk
+
+""" Helper functions """
+
+# Will attempt to do request "req1" with the optional parameters.
+# If fails it will attempt to do "req2" with the optional parameters.
+# If once of the request success this method will return True, if both fail- False.
+def helperFuncDo2Requests(req1, req2, is_verify=False, data=None, req_type="get"):
```

Comment:
What I saw in your usage of this function:
* You always use the default parameters
* req1 and req2 are actually the same, except for the protocol.

Take a look at this function:
```python
# just send the url without the http/s prefix
def do_request(url):
    try:
        results = map(lambda protocol: requests.get("{}://{}".format(protocol, url), verify=False), ["http", "https"])
        return (result for result in results if result.status_code==200).next()
    except:
        return False
```
I think it is far better to write short and understandable code, rather than a multipurpose helper function, which takes the attention away from the real Hunter. 
Though, looking at what Liz suggested about separating the code to insecure and secure can be somewhat more understandable and effective (less actual requests being sent). But in any case, a point to consider.